# CD7: clustering

In this notebook, we perform a clustering analysis of the CD7 data on features extracted from SCIP.

In [11]:
import pyarrow.parquet as pq
from ehv.correlation import Correlation

In [8]:
import os
os.environ["VSC_DATA_VO_USER"]

'/data/gent/vo/000/gvo00070/vsc42015'

In [9]:
df = pq.read_table("/data/gent/vo/000/gvo00070/vsc42015/datasets/CD7/results/scip/202112151458/features.parquet")

In [10]:
df.shape

(136951, 1290)

## Reduce input dimensionality

The original dataframe has 1290 features. This is too high for a clustering analysis. I will reduce the input dimensionality with a correlation analysis and a dimensionality reduction.

In [ ]:
corr = Correlation(0.8)